src/sentry/api/endpoints/organization_traces.py

```diff
@@ -131,6 +136,18 @@ def data_fn(offset: int, limit: int):
             snuba_params.start = min_timestamp - buffer
             snuba_params.end = max_timestamp + buffer
 
+            all_projects = self.get_projects(
+                request,
+                organization,
+                project_ids={-1},
+                project_slugs=None,
+                include_all_accessible=True,
```

Comment:
you technically don't need to set this to true since pasisng in -1 will set it https://github.com/getsentry/sentry/blob/824d4a8bb5e9e3ff06d1285fe53311911e187301/src/sentry/api/bases/organization.py#L370